src/components/MultiSelect/MultiSelect.stories.tsx

```diff
@@ -223,24 +220,19 @@ const meta: Meta<typeof MultiSelect> = {
     isSearchable: {
       table: { defaultValue: { summary: false } },
     },
-    items: {
-      control: false,
-    },
     listOverflow: {
       control: "radio",
       options: multiSelectListOverflowArray,
       table: { defaultValue: { summary: "scroll" } },
     },
-    onChange: {
-      table: { defaultValue: { summary: false } },
-    },
+    onChange: { control: false },
     onMixedStateChange: {
-      table: { defaultValue: { summary: false } },
+      control: false,
     },
     selectedItems: {
       control: false,
     },
-    width: {
+    multiSelectWidth: {
```

Comment:
From the other PR, I rather keep this is `width` since it does work and instead manually add the description here.